burn-import/src/burn/node/unsqueeze.rs

```diff
@@ -0,0 +1,96 @@
+use super::{Node, NodeCodegen};
+use crate::burn::{Scope, TensorType, ToTokens, Type};
+use burn::record::PrecisionSettings;
+use proc_macro2::TokenStream;
+use quote::quote;
+
+#[derive(Debug, Clone, new)]
+pub struct UnsqueezeNode {
+    pub input: TensorType,
+    pub output: TensorType,
+    pub axes: Vec<i64>,
+}
+
+impl<PS: PrecisionSettings> NodeCodegen<PS> for UnsqueezeNode {
+    fn output_types(&self) -> Vec<Type> {
+        vec![Type::Tensor(self.output.clone())]
+    }
+
+    fn input_types(&self) -> Vec<Type> {
+        vec![Type::Tensor(self.input.clone())]
+    }
+
+    fn forward(&self, scope: &mut Scope, node_position: usize) -> TokenStream {
+        let input = scope.tensor_use_owned(&self.input, node_position);
+        let output = &self.output.name;
+        let shape_values = &self.axes.to_tokens();
+        let new_dims = self.output.dim.to_tokens();
+
+        quote! {
+            let #output: Tensor<B, #new_dims> = #input.unsqueeze_dims(&#shape_values);
+        }
+    }
+
+    fn into_node(self) -> Node<PS> {
+        Node::Unsqueeze(self)
+    }
+}
+
+// #[cfg(test)]
+// mod tests {
+//     use burn::record::FullPrecisionSettings;
+
+//     use super::*;
+//     use crate::burn::{
+//         graph::BurnGraph,
+//         node::{test::assert_tokens, unsqueeze::UnsqueezeNode},
+//         TensorType,
+//     };
+
+//     #[test]
+//     fn test_codegen_nodes() {
+//         let mut graph = BurnGraph::<FullPrecisionSettings>::default();
+
+//         graph.register(UnsqueezeNode::new(
```

Comment:
Tests are commented out.

Reply:
Is there any reason to include test here, rather than strictly testing via the model?

Reply:
Yes, the plan is that code generation is independent from ONNX graph. The ONNX file testing is end-to-end.  We can add other computation graphs.

Reply:
I uncommented the test and fixed it. Confirmed it's working locally.